.github/ISSUE_TEMPLATE/release.md

```diff
@@ -52,11 +52,17 @@ Feel free to edit this release checklist in-progress depending on what tasks nee
       the [eclipse-edc-management-api.yaml file](https://github.com/sovity/edc-extensions/blob/main/docs/eclipse-edc-management-api.yaml).
     - [ ] Update the Postman Collection if required.
     - [ ] Merge the `release-prep` PR.
-- [ ] Wait for the main branch to be green.
-- [ ] Test the release `docker-compose.yaml` with `RELEASE_EDC_IMAGE=ghcr.io/sovity/edc-dev:latest`.
-    - [ ] Ensure with a `docker ps -a` that all containers are healthy, and not `healthy: starting` or `healthy: unhealthy`.
+- [ ] Wait for the main branch to be green. You can check the status in GH [actions](https://github.com/sovity/edc-extensions/actions).
+- [ ] Validate the image
+  - [ ] Pull the latest latest edc-dev image: `docker image pull ghcr.io/sovity/edc-dev:latest`.
+  - [ ] Check that your image was built recently `docker image ls | grep ghcr.io/sovity/edc-dev`.
+  - [ ] Test the release `docker-compose.yaml` with `EDC_IMAGE=ghcr.io/sovity/edc-dev:latest`.
+  - [ ] Ensure with a `docker ps -a` that all containers are healthy, and not `healthy: starting` or `healthy: unhealthy`.
 - [ ] Test the postman collection against that running docker-compose.
-- [ ] Create a release and re-use the changelog section as release description, and the version as title.
+- [ ] Tag the release
+  - [ ] Tag the current main with the release version in the format `vx.y.z` for instance `v1.2.3`
+  - [ ] Fetch the latest version `git fetch`, `git tag vx.y.z origin/main`, `git push origin :refs/tags/vx.y.z`
```

Comment:
this is incorrect.

you should select "create tag on release" on the "Create Release Page"

Reply:
Fixed.